cycler.py

```diff
@@ -329,10 +329,55 @@ def simplify(self):
         # I would believe that there is some performance implications
 
         trans = self._transpose()
-        return reduce(add, (cycler(k, v) for k, v in six.iteritems(trans)))
+        return cycler(**dict((k, v) for k, v in six.iteritems(trans)))
 
 
-def cycler(label, itr):
+def cycler(*args, **kwargs):
+    """
+    Create a new `Cycler` object from the combination of
+    positional arguments or keyword arguments.
+
+    cycler(arg)
+    cycler(label1=itr1[, label2=iter2[, ...]])
+
+    Form 1 simply copies a given `Cycler` object.
+    Form 2 composes a `Cycler` as an inner product of the
+    pairs of keyword arguments.
+
+    Parameters
+    ----------
+    label : str
+        The property key.
+
+    itr : iterable
+        Finite length iterable of the property values.
+
+    Returns
+    -------
+    cycler : Cycler
+        New `Cycler` for the given property
+
+    """
+    if args and kwargs:
+        raise TypeError("cyl() can only accept positional OR keyword "
+                        "arguments -- not both.")
+
+    if len(args) == 1:
+        if not isinstance(args[0], Cycler):
+            raise TypeError("If only one positional argument given, it must "
+                            " be a Cycler instance.")
+        return copy.copy(args[0])
```

Comment:
Probably want to do a keep copy, I can in to some really weird bugs earlier due to underlying dict objects getting shared.  I think this will play badly with the re-labeling functionality.